test/fixtures/templates.yml

```diff
@@ -93,3 +93,19 @@ locked:
   operatingsystems: centos5_3
   locked: true
   type: ProvisioningTemplate
+
+rendering_test_template:
```

Comment:
Use a factory instead of adding a new fixture per test. Simply `template = FactoryGirl.build(:provisioning_template, template: '<%= @host.info %>')` in the test.